### DISCUSSION TITLE
Using Nuxt Content with Django

### DISCUSSION THREAD

**Discussion by ghost** [nuxt/nuxt#16552] [Questions]
Hi everyone,
I'm fairly new to web development and I'd appreciate any help from people with more experience, especially when it comes to rendering modes.

I'm building a personal website with a small blog - although this could be done with a simple static site, I wanted to try something more advanced and extensible. Initially, I built the backend with Django and frontend with Vue.

I've "migrated" the frontend to Nuxt and wanted to take advantage of the Content's module - however, I'm not sure how to do this, since the blog posts are fetched from the Django backend. While I could eliminate the Django backend completely (or at least when it comes to the blog posts), I'd like to know if there's a "common way" of achieving this with this setup, e.g. outputing the backend's response to the content directory and going from there (which seems a bit clumsy but it's the only thing I can think of). 

In other words, how can I use the Nuxt Content module, when the content files aren't part of the Nuxt app, but are sent from a different API?
